blog-posts/blog4.qmd

```diff
@@ -4,3 +4,18 @@ description: ""
 date: "11/14/22"
 editor: visual
 ---
+
+My name is Josh Yamamoto and I'm finishing up my final year as a Mathematics & Statistics major at Reed College. I have been lucky enough to interact with data science in a wide variety of settings and flavors. Out of that experience grew my primary interest: a deep passion for propagating thoughtfulness and intentionality in all of the ways that data science gets done. While I have a special love for solving very technical problems, I care just as much about how those problems get solved.
+
+I have a lot of research experience having done social justice computer science research at Reed College, biostatistics research at a joint program at Duke & NC State, and most recently forestry data science research at Harvard University. Each of these programs have been both deeply technical and deeply collaborative, giving me a strong appreciation for how important partnership is to solving difficult problems. 
+
+I also have done work as a data science consultant at my college for the last three years, advising and implementing programs in R and other data technologies to catalyze research efforts by faculty and student researchers. I have also served as a course assistant and tutor for machine learning, and introductory statistics courses. While I always enjoy solving technical problems, I get just as much joy out of aiding and equiping other people to learn and solve problems themselves.
+
+You can check out my complete resume [here](resume.qmd)!
+
+I'm looking for a job at a place where I can be a part of building tools for data scientists and statisticians. 
```

Comment:
```suggestion
I'm looking for a role where I can be a part of building tools for data scientists and statisticians. 
```

Reply:
This ought to come before the resume, and be a bit longer. I'm guessing you'd like to work remotely or in the Portland, OR area? How can folks get in touch with you? Are you open to chatting with folks starting now, and when is your desired start date (e.g. "June 2023")?

Also, "for" or "with" data scientists and statisticians?